### ISSUE TITLE
As Forked computatoins are killed do we need EventSource any more?

### ISSUE THREAD

**Issue by safareli** [slamdata/purescript-halogen#570]
So basically one can fork computation which calls `eval` with some query instead of using subscription. subscription can terminate by itself, but with some helper functions it could be done for forked computations too. so there is no clear benefit for using subscription over forking (is there any?) 

If we can emulate subscription by fork then I suppose we can remove `Subscribe` node from `HalogenF`. As this is the only node which uses `act` type variable of `HalogenF` it means we can remove that variable too and we would have one less type variable for HalogenM?

**Comment by garyb** [comment]
I think, of the two, I'm more likely to remove forking rather than event sources - I did consider just doing it in the current master, but I figured I'd see how annoying it would be to not be able to fork, by using ES anywhere I usually would have forked.

Aside from being able to terminate themselves, it's much easier to have an event listener / callback raise messages via ES rather than doing something in a fork. I guess it's still possible in a fork, but you'd have to jump through a bunch of `AVar` hoops to be able to do something similar.
